Document known issue with Kibana 8.11.2 using secure settings (#7373)
elastic/cloud-on-k8s: docs/release-notes/highlights-2.10.0.asciidoc

[[release-highlights-2.10.0]]
== 2.10.0 release highlights

[float]
[id="{p}-2100-known-issues"]
=== Known issues

> - Upgrading Kibana configured with <<{p}-kibana-secure-settings,secure settings>> to version `8.11.2` lead to Kibana pods in Error

Reviewer: ```suggestion
- Upgrading Kibana configured with <<{p}-kibana-secure-settings,secure settings>> to version `8.11.2` leads to Kibana pods being unavailable with status `Init:Error`.
```
